src/cclib/parser/utils.py

```diff
@@ -61,11 +61,9 @@ def symmetrize(m, use_triangle='lower'):
 
 def convertor(value, fromunits, tounits):
     """Convert from one set of units to another.
-
```

Comment:
Don't remove random blank lines here and elsewhere.

Reply:
Sorry, just fixing. Thanks:)